doc/pygambit.user.rst

```diff
@@ -183,6 +183,22 @@ the top level index is the choice of the first player, the second level index of
 and so on.  Therefore, to create a two-player symmetric game, as in this example, the payoff matrix
 for the second player is transposed before passing to :py:meth:`.Game.from_arrays`.
 
+There is a reverse function :py:meth:`.Game.to_arrays` that produces
+the players' payoff tables given a strategic game. The output is the list of numpy arrays,
+where the number of produced arrays is equal to the number of players.
+
+.. ipython:: python
+
+   m, m_transposed = g.to_arrays()
+   m
+
+There is a parameter `dtype`` passed to :py:meth:`.Game.to_arrays` that configures the data type of
+payoffs in the generated arrays. Supported types are :py:class:`.Rational` (default) and float.
```

Comment:
Actually this isn't quite true - we support any type which can convert from a Python `Fraction`.  Specifically for example, `str` works just fine!  And in fact that could be a nice example to add as well in addition to the `float`.